tests/various/param_struct.ys

```diff
@@ -0,0 +1,25 @@
+read_verilog -sv -dump_ast1 -dump_ast2 << EOF
+package p;
+typedef struct packed {
+  logic a;
+  logic b;
+} struct_t;
+
+parameter struct_t c = {1'b1, 1'b0};
+endpackage
+
+module dut ();
+parameter p::struct_t d = p::c;
+
+logic e = d.a;
+logic f = d.b;
```

Comment:
To have this test verify that `d.a`, etc., can be used in constant expressions, let's change `logic` to `localparam`.

Reply:
Done.